config/resources/domain.lisp

```diff
@@ -17,9 +17,9 @@
 
 (define-resource export ()
   :class (s-prefix "export:Export")
-  :properties `((:filename :string ,(s-prefix "nfo:filename"))
+  :properties `((:filename :string ,(s-prefix "nfo:fileName"))
                 (:format :string ,(s-prefix "dct:format"))
                 (:filesize :number ,(s-prefix "nfo:fileSize"))
                 (:created :datetime ,(s-prefix "dct:created")))
-  :resource-base (s-url "http://data.lblod.info/id/exports/")
+  :resource-base (s-url "http://mu-exporter/exports/")
```

Comment:
This change is not strictly needed. `:resource-base` only affects the resources created by `mu-cl-resources` itself, but the exported files are created by the `mandaten-download-generator-service` and never by `mu-cl-resources` so this is never used.